<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingView</class>
 <widget class="QWidget" name="TestingView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>951</width>
     <height>831</height>
    </rect>
   </property>
   <widget class="QLabel" name="testTitle">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Результаты тестирования</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>471</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: black;
background-color: rgba(239, 240, 242, 0.8);</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>411</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: transparent;
padding-left: 25px;</string>
          </property>
          <property name="text">
           <string>Часть тестовой выборки:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="partTest">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(222, 227, 231);
border-radius: 5px;
margin-right: 25px;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="downloadTest">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>411</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
border-radius: 10px;
background-color: white;
color: black;
}

QPushButton:hover {
background-color: black;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Загрузить test.сsv </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startTest">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>411</width>
          <height>61</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgba(222, 227, 231, 0.5);
color: white;
margin: 10px 0 0 0;</string>
        </property>
        <property name="text">
         <string>Запустить тестирование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>411</width>
          <height>61</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
margin: 10px 0 0 0;
border-radius: 10px;
background-color: white;
color: black;
}

QPushButton:hover {
margin: 10px 0 0 0;
background-color: black;
color: white;
}</string>
        </property>
        <property name="text">
         <string>Сброс настроек тестирования</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="experiment">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>61</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>411</width>
            <height>61</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background-color: rgba(222, 227, 231, 0.5);
color: white;
margin: 10px 0 0 0;</string>
          </property>
          <property name="text">
           <string>Провести тестирование(N раз)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="numberExperiments">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(222, 227, 231);
border-radius: 5px;
margin-right: 25px;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>-2.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="paintTitle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Рисовать здесь</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>851</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: black;
background-color: rgba(239, 240, 242, 0.8);
/*background-color: rgba(221, 222, 222, 0.2);*/</string>
    </property>
    <widget class="QLabel" name="resTitle">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string>Результат:</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="paint">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black;
border-radius: 20px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cleanPaint">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>190</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>61</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 10px;
background-color: white;
/*background-color: rgb(245, 49, 49);
color: white;*/
color: black;
margin: 10px 0;
}

QPushButton:hover {
background-color: black;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadImg">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>190</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>61</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 10px;
background-color: white;
/*background-color: rgb(245, 49, 49);
color: white;*/
color: black;
margin: 10px 0;
}

QPushButton:hover {
background-color: black;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Загрузить картинку</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startTestIMG">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>190</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>61</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 10px;
background-color: white;
/*background-color: rgb(245, 49, 49);
color: white;*/
color: black;
margin: 10px 0;
}

QPushButton:hover {
background-color: black;
color: white;
}</string>
     </property>
     <property name="text">
      <string>Обработать</string>
     </property>
    </widget>
    <widget class="QLabel" name="result">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>70</y>
       <width>231</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>351</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
color: black;
background-color: rgba(239, 240, 242, 0.8);
padding: 20px;</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>293</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="fMeasure">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
padding: 0;</string>
        </property>
        <property name="text">
         <string>f-мера:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="recall">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
background-color: transparent;</string>
        </property>
        <property name="text">
         <string>Полнота:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="precision">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
background-color: transparent;</string>
        </property>
        <property name="text">
         <string>Прецизионность:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="time">
        <property name="minimumSize">
         <size>
          <width>27</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
background-color: transparent;</string>
        </property>
        <property name="text">
         <string>Затраченное время:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="accuracy">
        <property name="minimumSize">
         <size>
          <width>188</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>188</width>
          <height>37</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
padding: 0;</string>
        </property>
        <property name="text">
         <string>Средняя точность:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="valueAccuracy">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
border: none;</string>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="valuePrecision">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
border: none;</string>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="valueRecall">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
border: none;</string>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="valueFMeasure">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
border: none;</string>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="valueTime">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 0;
border: none;</string>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
